telegram/__init__.py

```diff
@@ -215,6 +219,28 @@ def parse_item(self, item):
             if value in ['true', 'True', '1']:
                 self._items_text_message.append(item)
             return self.update_item
+        
+        if self.has_iattr(item.conf, ITEM_ATTR_CONTROL):
+            attr = self.get_iattr_value(item.conf, ITEM_ATTR_CONTROL)
+            #self.logger.debug("parse item: {0} {1}".format(item, key))
+            name = item.id()    # default
+            type = 'toggle'     # default
```

Comment:
`type` can be used to check the type of a python object. Although there is another function isinstance() you should not use type here.